doma-processor/src/main/java/org/seasar/doma/internal/apt/annot/ModifyAnnot.java

```diff
@@ -103,6 +109,14 @@ public AnnotationValue getSqlLog() {
     return sqlLog;
   }
 
+  public DuplicateKeyType getDuplicateKeyType() {
```

Comment:
We have a rule to add the `Value` suffix to method names that get resolved values of `AnnotationValue`.
Therefore, please change the name to `getDuplicateKeyValue`.